src/Sql/PostgreSQL/Functions/user_update.sql

```diff
@@ -0,0 +1,76 @@
+drop function user_update(UUID,VARCHAR,VARCHAR,INT4,VARCHAR,VARCHAR,VARCHAR,VARCHAR,TEXT,VARCHAR,TEXT,TEXT,TIMESTAMPTZ,TEXT,TEXT,TEXT,INT4,TEXT,TEXT,INT8,INT2,INT2,VARCHAR,VARCHAR,VARCHAR,INT4,INT4,TIMESTAMPTZ,TIMESTAMPTZ);
+
+CREATE OR REPLACE FUNCTION user_update 
+(
+    _id									UUID,
```

Comment:
formatting seems weird in this file. Looks like tabs mixed with spaces. should just be spaces.